app/views/events/show.html.erb

```diff
@@ -2,27 +2,50 @@
 <br>
 <%= @event.description %>
 <br>
-<% @event.reviews.each do |review| %>
-  <%= review.user.email %> <br>
-  <%= review.content %> <br>
-  <%= review.stars %> <br>
-  <%if current_user == review.user %>
-    <%= link_to "Modifier mon commentaire", edit_event_review_path(@event, review) %>
-    <br>
-    <%= link_to "Supprimer mon commentaire", event_review_path(@event, review), method: :delete, data: { confirm: "Supprimer mon commentaire ?" } %>
-    <% end %> <br>
+<% @star_total = 0 %>
+<% if @event.reviews.count > 0 %>
+  <% @event.reviews.each do |review| %>
+    <% @star_total += review.stars.to_i %>
+  <% end %>
+  <% @star_average = @star_total.div(@event.reviews.count).round%>
```

Comment:
This calculation should be refactor in the model so you can call something simpler in the view